DESCRIPTION

```diff
@@ -19,11 +23,34 @@ Encoding: UTF-8
 LazyData: true
 biocViews: Software, Infrastructure
 Imports:
-    Biostrings
+    Biostrings,
+    utils,
+    stats,
+    xlsx,
+    readxl,
+    visNetwork,
+    dplyr,
+    biomaRt,
+    ggplot2,
+    biogridr,
+    devtools
 Suggests:
     testthat,
     BiocStyle,
     knitr,
     rmarkdown
+Remotes: NElnour/biogridr
```

Comment:
Oh no ... that's an external dependency. We really should not do that. Why do you you need this package?

Reply:
Hello, 

biogridr allowed me to get genetic interaction data from BioGrid before fetchData was available. I'll remove it.